src/main/java/uk/gov/di/ipv/lambda/CredentialIssuerHandler.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.di.ipv.lambda;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.oauth2.sdk.util.StringUtils;
+import uk.gov.di.ipv.domain.ErrorResponse;
+import uk.gov.di.ipv.helpers.ApiGatewayResponseGenerator;
+import uk.gov.di.ipv.helpers.RequestHelper;
+import uk.gov.di.ipv.helpers.ResponseBodyHelper;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+public class CredentialIssuerHandler implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+
+    private Set<String> validCredentialIssuers = Set.of("PassportIssuer", "FraudIssuer");
+
+    @Override
+    public APIGatewayProxyResponseEvent handleRequest(APIGatewayProxyRequestEvent input, Context context) {
+
+        Map<String, String> body = ResponseBodyHelper.parseRequestBody(input.getBody());
+        String authorizationCode = body.get("authorization_code");
+        String credentialIssuer = body.get("credential_issuer_id");
```

Comment:
Could these keys be defined as enums somewhere?

Reply:
We added a CredentialIssuerRequestDto to model the request payload

Reply:
I guess what I was getting at was trying to have those strings only defined in one place so only needs changing once if they ever change. In the `CredentialIssuerRequestDto` they're now defined in two places. If you defined an enum and then used it in those annotations you can alter the actual value more easily if you need to.